### DISCUSSION TITLE
Generate OTP from Auth.Admin

### DISCUSSION THREAD

**Discussion by timdiacon** [supabase/supabase#12286] [Feature Requests]
It would be great if there was a`supabase.auth.admin.generateOtp` method.

Our use case it to be able to send OTP codes to users via WhatsApp however it would also allow the user of other SMS providers other than just Twilio / Message Bird / Vonage

**Comment by thorwebdev** [comment]
Hey @timdiacon, thanks for your feedback! This work is being tracked here: https://github.com/supabase/gotrue/issues/385
